src/components/elements/Table.js

```diff
@@ -40,24 +40,43 @@ const TableStyle = styled.table`
     padding-bottom: 10px;
   }
 
-  & tr:nth-child(even) td {
-    background-color: ${greyScale(9)};
-  }
 `;
 
-export const TableCell = ({ children, header }) => (
+export const TableCell = ({ children, header, }) => (
   <>{header ? <th>{children}</th> : <td>{children}</td>}</>
 );
 
-export const TableBody = styled.tbody``;
+export const TableSpanCell = styled.td`
+  grid-column: 1 / span 5;
+`
+
+export const TableRightAlignCell = styled.td`
+  text-align: right;
+  margin-right: 50px;
```

Comment:
Using text-align + margin-right shouldn't be necessary since we're using css grid. It doesn't look like this styled component is being used anyway, so can you remove it?

Reply:
You marked as resolved but didn't change it. Was this intentional?